Github pages fix

elboyran opened the pull request:
The [github page](https://dianna-ai.github.io/dianna/) does not render properly when using summary + details construct. 
this PR is an attempt to fix that.

elboyran committed:
Update README.md

elboyran committed:
Update README.md

added extra empty line

elboyran committed:
Update README.md

added more lines

egpbos commented:
@copilot The deployment on Pages shows that the summary/details "eats up" all the content that comes after it, so it seems like somehow the closing tag is not interpreted correctly. Please analyse.

Copilot committed:
Fix details blocks: remove unclosed p tags causing content to be eaten up

Agent-Logs-Url: https://github.com/dianna-ai/dianna/sessions/2d215069-4978-4c83-b3b4-8b5f61663575

Co-authored-by: egpbos <6146598+egpbos@users.noreply.github.com>

egpbos reviewed (approved):
LGTM!
